README.md

```diff
@@ -1,9 +1,14 @@
 # School Library
 
```

Comment:
Please add instructions to run the app locally. It is important to add installation instructions for the user who is not a programmer.